research/upgrade_interviews/index.md

```diff
@@ -0,0 +1,56 @@
+---
+parent: Research
+
+---
+# Upgrade Interviews
+
+*Date: July 2020*
+
+*Note: this is an article about a research activity on how users upgrade OpenShift. If you're looking for documentation about how to upgrade OpenShift, please go to [this link.](https://docs.openshift.com/container-platform/4.4/welcome/index.html)*
+
+## Background
+
+When considering major design changes to how users can upgrade their clusters in OpenShift 4, we wanted to be sure we understood when, why, and how our users upgrade. We interviewed 1 internal team and 5 customers. We had a mix of those running 4.x in production and still running 3.11 with 4.x in testing clusters. We used a semi-structured interview approach to allow room for us to explore important issues to our users we weren’t already aware of. Here are a few example questions from our interview guide.
+
```

Comment:
I'd suggest slight rewording of the first sentence. It was a little hard for me to parse. What about something like:

"When considering design changes to the upgrade experience in OpenShift 4, we wanted to be sure we understood when, why, and how our users upgrade first."

Reply:
Good point, I'll adjust that.